Makefile

```diff
@@ -0,0 +1,51 @@
+#==================
+# Macros
+#==================
+
+pardir = $(patsubst %/,%,$(dir $(abspath $(1))))
+
+
+#==================
+# Helper Constants
+#==================
+
+THIS := $(abspath $(lastword $(MAKEFILE_LIST)))
+HERE := $(call pardir,$(THIS))
+DRAW := $(HERE)/draw_party
+
+
+#==================
+# Targets
+#==================
```

Comment:
Add a `package` target that zips up what we need to deploy to beanstalk.

Reply:
I think I'll leave that for when we make this beanstalk deployable, this portion is really set up and verify that you can run it locally, even though I did add one elasticbeanstalk config file, that was more of a note